### ISSUE TITLE
Scrum Report #2

### ISSUE THREAD

**Issue by Landseer100** [ptellier/nimbase#4]

**Comment by jakemilad** [comment]
b2c3e

I've worked on configuring the routing of pages in the project using React router dom, also, I've built the template for the login page as well as the signup page, which has all the necessary elements needed to implement the sign-in and authentication. No particular blockers for this week.

**Comment by Landseer100** [comment]
I’ve been very busy with moving and so I have not had much capacity to work on my portion of the project this sprint. But I am flying home tonight and will be finishing my tasks within the next week. So far no blockers, still working on implementing the initial design mocks.

**Comment by Aryan-B** [comment]
Bhairaw Aryan (k0i3b)

I worked on building a backup app over the past two weeks, focusing on setting up the backend, frontend, and database. The project was based on a previously proposed idea. The major challenge I faced was integrating Mapbox GL into the app, as it required going through the documentation and understanding how to incorporate it effectively. For the upcoming iteration, my plan is to complete the backup app by the end of this week. Following that, I will shift my focus to the development operations (DevOps) aspect of the actual Nimbase project.

**Comment by ptellier** [comment]
Phillip Tellier (i3b6m)

I created a back-end express API that communicates with mongodb to do CRUD operations with users or projects (in separate collections). I created a "Query" class in the front-end that does all the basic HTTPS requests required to fetch data from the back-end using the axios package. Things went smoothly for the most part, my only blockers are my current lack of understanding about security standards for authentication which will require some studying before I can help out with implementing the user authentication. For the next two weeks, I will look more into user authentication, and help out with hooking up the "project edit" and "project explore" pages to the mongodb database.
